Directory.Packages.props

```diff
@@ -53,22 +56,25 @@
           even during major version bumps, so compatibility is not a concern here.
     -->
     <PackageVersion Include="System.Diagnostics.DiagnosticSource" Version="9.0.0-rc.1.24431.7" />
+  </ItemGroup>
 
-    <!-- A conservative version of System.Text.Encodings.Web must be used here since there is no backward compatibility guarantee during major version bumps. -->
+  <ItemGroup>
+    <!-- We use conservative versions of these packages for older runtimes where
+    an upgrade might introduce breaking changes. For example see:
+    https://learn.microsoft.com/dotnet/core/compatibility/serialization/7.0/reflection-fallback#type-of-breaking-change.
```

Comment:
FYI @CodeBlanch if you need a better link, this might be one https://devblogs.microsoft.com/dotnet/system-text-json-in-dotnet-7/#breaking-changes

Reply:
Updated thanks!